sprint1/style.css

```
@@ -0,0 +1,283 @@
+/* 
+  2025.04.01
+  스프린트 미션 1
+  박연희
+*/
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+}
+a {
+  color: inherit;
+}
+
+/* 폰트 */
+@font-face {
+  font-family: 'Pretendard-Regular';
+  src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
+
+/* 버튼 btn */
+.btn {
+  cursor: pointer;
+  display: inline-block;
+  text-align: center;
+  white-space: nowrap;
+}
+.btn_large {
+  padding: 12px 124px;
+  border-radius: 40px;
+  font-size: 20px;
+}
+.btn_medium {
+  padding: 11px 71px;
+  border-radius: 40px;
+  font-size: 18px;
+}
+.btn_small {
+  padding: 8px 23px;
+  border-radius: 8px;
+  font-size: 16px;
+}
+.btn-color1 {
+  color: #ffffff;
+  background-color: #3692FF;
+}
+
+/* 기본넓이 */
+.default-width {
+  max-width: 1920px;
+  margin: 0 auto;
+  transition: 0.5s;
+}
+
+/* body */
+body {
+  font-family: 'Pretendard-Regular';
+  background-color: #ffffff;
+  color: #374151;
+  font-weight: 400;
+}
+
+/* header */
+.header-inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 70px;
+}
+.logo-mobile {
+  display: none;
+}
```

개인의 선택이긴 한데, 미디어쿼리를 스타일 블록마다 작성해주는 방식이 있고 지금처럼 한꺼번에 아래에서 작성해주는 방식이 있는데, 저는 전자를 선호하긴합니다 ㅎㅎ 이런 방식도 있다는걸 참고해두시면 좋을것같아요 :)
 
```suggestion
.logo-mobile {
  display: none;
  @media (max-width: 375px) {
    display: block;
  }
}
```